src/app/services/food-description/food-description.service.ts

```diff
@@ -13,7 +13,7 @@ const httpOptions = { headers: new HttpHeaders({'Content-Type': 'application/jso
 
 export class FoodDescriptionService {
 
-  endpoint = environment.foodServiceUrl + '/ffq';
+  endpoint = 'https://ffq-food-item-service.azurewebsites.net/ffq'; ///fooddescription/update/
```

Comment:
Hey Greg keep that environment.foodServiceUrl variable this one can be changed in the environments folder from one area so we dont hard code the url into the code